nodes/quantum/classical-register/classical-register.js

```diff
@@ -14,6 +14,7 @@ module.exports = function(RED) {
     this.name = config.name.trim().toLowerCase().replace(/ /g, '_');
     this.classicalBits = config.classicalBits;
     const node = this;
+    let script = '';
```

Comment:
This line needs to be moved to inside the input listener, otherwise it won't be reset between runs.